api.py

```diff
@@ -8,163 +8,49 @@
-    # https://docs.steamship.com/workspaces/data_model/index.html
-    file = File.create(self.client,
-                       blocks=[Block.CreateRequest(text=prompt_text)])
-
-    # This requests generation from the parameterized prompt. Tagging with our prompt generator
-    # plugin will result in a new tag that contains the generated output.
-    # We `wait()` because generation of text is done asynchronously and may take a few moments
-    # (somewhat depending on the complexity of your prompt).
-    file.tag(plugin_instance=generator.handle).wait()
-
-    # Retrieve the stored generated content and return it in the body of the response.
-    # Refreshing the file ensures that we get the latest content for the file.
-    return self._generated_text(file.refresh())
-
-  def _generated_text(self, prompt_file: File) -> str:
-    """Get the generated text for a prompt file."""
-
-    # Here, we iterate through the content blocks associated with a file
-    # as well as any tags on that content to find the generated text.
-    #
-    # The Steamship data model provides flexible content organization,
-    # storage, and lookup. Read more about the data model via:
-    # https://docs.steamship.com/workspaces/data_model/index.html
-    for text_block in prompt_file.blocks:
-      for block_tag in text_block.tags:
-        if block_tag.kind == TagKind.GENERATION:
-          return self._sanitize(block_tag.value[TagValue.STRING_VALUE])
-
-    return ""
-
-  def _sanitize(self, text: str):
-    """Remove any leading/trailing whitespace and partial sentences.
-
-      This assumes that your generated output will include consistent punctuation. You may
-      want to alter this method to better fit the format of your generated text.
-      """
-    last_punc = -1
-    for i, c in enumerate(reversed(text)):
-      if c in ".!?\"":
-        last_punc = len(text) - i
-        break
-    if last_punc != -1:
-      result = text[:last_punc + 1]
-    else:
-      result = text
-    return result.strip()
+    # Modify this to customize behavior to match your needs.
+    PROMPT = "Say an unusual greeting to {name}. Compliment them on their {trait}."
+
+    # When this package is deployed, this annotation tells Steamship to expose an endpoint that
+    # accepts HTTP POST requests for the `/generate` request path.
+    # See README.md for more information about deployment.
+    @post("generate")
+    def generate(self, **prompt_args) -> str:
+        """Generate text from prompt parameters."""
+        llm_config = {
+            "max_words": 30,    # Controls length of generated output.
+            # Controls randomness of output (range: 0.0-1.0).
+            "temperature": 0.8
+        }
+        llm = self.client.use_plugin("gpt3", config=llm_config)
+        return llm.generate(self.PROMPT, prompt_args)
 
 
 # Try it out locally by running this file!
+# You'll need to set the STEAMSHIP_API_KEY environment variable
```

Comment:
I think both comments or __main__ print statement are both fine. Thinking out loud -- a nearly LoC-neutral change would be instead of the comments:

if __name__ == "__main__":
  if not os.environ.get('STEAMSHIP_API_KEY'):
    print("Please set the STEAMSHIP_API_KEY environment variable. You can get one at <url>") 
    exit(-1)



Also we could put a:

```Steamship.check_environment(type="replit", exit=True)```

helper in?